lib/operators/param.js

```diff
@@ -1,7 +1,7 @@
 var errors = require("../errors");
 
 module.exports = {
-  parse: function(key, dflt) {
+  parse: function() {
```

Comment:
Is there an option to leave these alone? I quite like what's passed to the function to be in the definition.